src/index.rs

```diff
@@ -1336,7 +1333,7 @@ mod tests {
                 .update_all()
                 .expect("Should update index correctly");
 
-            assert_eq!(initial_index , index_update_all);
+            assert!(initial_index != index_update_all);
```

Comment:
`assert_ne!`

Reply:
updated like this code.
assert_ne!(
                initial_index, index_update_all,
                "we are testing that the values are not equal"
            );